### DISCUSSION TITLE
Jekyll Plug-in for consideration

### DISCUSSION THREAD

**Discussion by pippim** [jekyll/jekyll#8895] [Ideas]
The plug-in: [insert_git_code.rb](https://blog.bryanroessler.com/2020-03-18-insert-code-blocks-from-repository/) is a simple ruby gem invoked by a simple liquid include.

It saves you from having to copy your source code from your repo into your blog. All too often documentation falls out of sync with today's source code changes.

Kindly review the plug-in and include ASAP if you agree it is a simple, secure and powerful plug in for G-H Pages.

**Comment by mattr-** [comment]
We have no control over what gets included in the GitHub Pages product.

**Reply by pippim** [reply]
@mattr-  My apologies for thinking this was the place to post. Do you know a suitable place to post new features requests?

**Reply by ashmaroli** [reply]
@pippim To try and enhance the GitHub Pages experience, please file a ticket at https://github.com/github/pages-gem

**Reply by pippim** [reply]
@ashmaroli New ticket filed: [New Gem - Insert github repo code into Github Pages #804](https://github.com/github/pages-gem/issues/804).

Please up-vote the request if you think it will help yourself or others :)
